### PR TITLE
report(redesign): runtime settings footer

### PR DESCRIPTION
**Pull request by Hoten**
messed up #8192

### CODE REVIEW

**Comment by Hoten** [comment]
this was a silly time. Should be good now.

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -98,13 +101,14 @@
   --color-red-700: #EB0F00;
   --color-red: #FF4E42;
   --color-white: #FFFFFF;
+  --env-name-min-width: 220px;
   --header-padding: 20px 0 20px 0;
+  --item-bg: var(--color-black-100);
+  --item-padding: 10px 0px;
```
Comment (paulirish) [inline lighthouse-core/report/html/report-styles.css]: i presume `item` isn't specific to the env stuff?

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: just checked, it's only uses for the env table. renamed.

**Comment by Hoten** [comment]
brb deleting `git-rebase` from my machine

File: lighthouse-core/report/html/report-styles.css [hunk 2]
```diff
@@ -83,6 +83,9 @@
   --circle-size: calc(3 * var(--header-font-size));
 
   /* TODO(cjamcl) clean up unused variables. */
+  --audits-margin-bottom: 40px;
+  --circle-size-big: 120px;
+  --circle-size: 96px;
```
Comment (patrickhulce) [inline lighthouse-core/report/html/report-styles.css]: ~~what are these doing here/should we remove the `circle-size` above it?~~

oh looks like they're just alphabetized now :) still remove the one duplicate above it?

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: done

**Comment by Hoten** [comment]
linking to #8185

**Comment by Hoten** [comment]
@paulirish look good?

**Comment by paulirish** [comment]
Yup

**Pull request merged**